modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldServiceTest.java

```diff
@@ -124,8 +138,70 @@ public void testAddCustomObjectField() throws Exception {
 						"permission for")));
 		}
 
+		try {
```

Comment:
@MarinhoFeliphe do we need to cover this scenario in ObjectFieldServiceTest? What do you think about it?